docs/resources/bootstrap_git.md

```diff
@@ -30,6 +30,7 @@ resource "flux_bootstrap_git" "this" {
 - `components` (Set of String) Toolkit components to include in the install manifests. Defaults to `[source-controller kustomize-controller helm-controller notification-controller]`
 - `components_extra` (Set of String) List of extra components to include in the install manifests.
 - `disable_secret_creation` (Boolean) Use the existing secret for flux controller and don't create one from bootstrap
+- `delete_git_manifests` (Boolean) Delete manifests from git repository.
```

Comment:
Can you please add a `Defaults to true` at the end?

Reply:
Done. But defaults to `false`) - don't delete git manifests(current provider behavior).